runtime/lib/registry/Registry.cpp

```diff
@@ -15,8 +33,66 @@ namespace py = pybind11;
 // https://pybind11.readthedocs.io/en/stable/advanced/misc.html#common-sources-of-global-interpreter-lock-errors
 std::unordered_map<int64_t, py::function> *references;
 
+std::string libmlirpath;
+
+void convertResult(py::handle tuple)
+{
+
+    py::object unrankedMemrefPtrSizeTuple = tuple.attr("__getitem__")(0);
+
+    py::object unranked_memref = unrankedMemrefPtrSizeTuple.attr("__getitem__")(0);
+    py::object element_size = unrankedMemrefPtrSizeTuple.attr("__getitem__")(1);
+    py::object unranked_memref_ptr_int = unranked_memref.attr("value");
+
+    void *unranked_memref_ptr = (void *)(py::cast<long>(unranked_memref_ptr_int));
```

Comment:
```suggestion
    void *unranked_memref_ptr = reinterpret_cast<void *>(py::cast<long>(unranked_memref_ptr_int));
```
Assuming we are in the CPP layer, we should avoid C-style casts.